data-structures/README.md

```diff
@@ -0,0 +1,5 @@
+# Data structures
+
+### c or c++
+
+1. [Singly Linked List](c-or-cpp/SinglyLinkedList.cpp)
```

Comment:
After the folder name is changed to **linked lists** this can be changed to **stack implementation** (or something similar) because it is [stack](https://en.wikipedia.org/wiki/Stack_(abstract_data_type)) data structure that you are implementing here using linked list.